src/porepy/numerics/contact_mechanics/contact_conditions.py

```diff
@@ -544,16 +548,18 @@ def _sliding_coefficients(
         coeff_M = self._M(Tt, cut, bf)
 
         # Regularization during the iterations requires computations of parameters
-        # alpha, beta, delta
-        alpha = -Tt.T.dot(-Tt + cut) / (self._l2(-Tt) * self._l2(-Tt + cut))
-        # Parameter delta.
-        # NOTE: The denominator bf is correct. The definition given in Berge is wrong.
-        delta = min(self._l2(-Tt) / bf, 1)
-
-        if alpha < 0:
-            beta = 1 / (1 - alpha * delta)
-        else:
-            beta = 1
+        # alpha, beta, delta. In degenerate cases, use
+        beta = 1
+        # Avoid division by zero:
+        l2_Tt = self._l2(-Tt)
+        if not np.isclose(l2_Tt, 0):
```

Comment:
Vil denne sjekken tilsvare det du fjerner lengre oppe?
Burde du sjekke om denne er nærme self.tol eller stemmer det med 0?

Reply:
Nei, den skal bare unngå å dele på null lokalt i beregningen av beta.
self.tol er et godt spørsmål. Svaret er nok sikkert ja, for konsistens.